src/util/object/formatters/recursive-data-converter.ts

```diff
@@ -2,22 +2,26 @@ export const recursiveDataConvertFilterLayer = (
   data: any,
   formatter: (params: any) => any
 ) => {
-  if (Array.isArray(data))
+  if (Array.isArray(data)) {
     return data.map((item) => {
+      // TODO: this condition is correct? This should get instanceof item or data?

```

Comment:
The validation is correct; it's to prevent objects of type `Date` from being modified